flatpak/org.nanuc.Axolotl.yml

```diff
@@ -1,12 +1,14 @@
 app-id: org.nanuc.Axolotl
 branch: main
-runtime: org.freedesktop.Platform
-runtime-version: '20.08'
-sdk: org.freedesktop.Sdk
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
 sdk-extensions:
   - org.freedesktop.Sdk.Extension.golang
   - org.freedesktop.Sdk.Extension.node12
 command: axolotl
+base: io.qt.qtwebengine.BaseApp
```

Comment:
If someone just builds the flatpak without intending to use it with qt, then these changes will just make the application bigger without adding any extra value, right?

I would suggest we split up the flatpak builds and have one which is the current, default manifest (axolotl-web) - and then we add a qt specific one for the qt install option. 

That would mean a bit of copy-paste and overhead of keeping the files in sync but I don't see these files changing too much.

Looking at the changes in this MR, that would lead us to two .desktop files, and two manifests.

axolotl/flatpak/ (here appdata.xml and Axolotl.png stays)
axolotl/flatpak/qt/ (here goes new .desktop and manifest)
axolotl/flatpak/web/ (here goes old .desktop and manifest)

Then the paths used in the manifests needs to be slightly adjusted, but that should be doable.

The benefit of this is also that if we want further builds in the future, it would just be one sub-folder away.

Reply:
That was one of the thoughts I had when I wrote it the first time. I agree that this is the better approach, though the other downside is in distributing it, as there are now two different versions of the Axolotl flatpak to host/inform users about. 

This is now implemented. Still need to fix finding the `guis` folder. That should be the last item.